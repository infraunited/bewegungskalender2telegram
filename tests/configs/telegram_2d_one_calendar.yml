output:
  type: 'telegram'
  stdout: true
caldav:
  url: 'http://127.0.0.1:18476'
  calendars:
    - url: '/d/5cbae8f0-5881-c7a2-4cbb-c5c1ba05bc25/'
  username: 'x'
  password: 'x'
  offset_days: 1
  number_days: 2
  current_day_override: "2022-12-31"
format:
  time_locale: "de_DE.utf8"
  timezone: "Europe/Berlin"
  show_calendar_name: false
  show_location: true
  show_description: false
  cutoff_description: 0
  header_txt: ""
  footer_txt: "here is the footer"
