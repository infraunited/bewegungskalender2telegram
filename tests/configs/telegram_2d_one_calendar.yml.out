📅 *__Sonntag__ 2023\-01\-01*

*Multi\-All\-Day*
🌍 _with location_

*Single All\-Day*


📅 *__Montag__ 2023\-01\-02*

*Multi\-All\-Day*
🌍 _with location_

__10:00__ *\-\>test \[s@\_\+\}*
🌍 _\]\|🔋/\~ \*&^'_


here is the footer